ci/docs/build.sh

```diff
@@ -25,10 +25,43 @@ nvidia-smi
 
 gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
-conda activate rapids
+
+conda install -c conda-forge conda-build -y
+
+# We don't use 'rapids' conda environment here.
+# cuCIM depends on CuPy 9 which should be adopted in 0.20
+LIBCUCIM_BLD_PATH=$WORKSPACE/ci/artifacts/cucim/cpu/.conda-bld
+CUCIM_BLD_PATH=/opt/conda/envs/rapids/conda-bld
+mkdir -p ${CUCIM_BLD_PATH}
+
+gpuci_conda_retry build -c ${LIBCUCIM_BLD_PATH} -c conda-forge/label/cupy_rc -c conda-forge -c rapidsai-nightly \
+    --python=${PYTHON_VER} \
+    --dirty \
+    --no-remove-work-dir \
+    --croot ${CUCIM_BLD_PATH} \
+    conda/recipes/cucim
+
 
 # TODO: Move installs to docs-build-env meta package
-gpuci_conda_retry install -c conda-forge markdown beautifulsoup4 jq sphinx-markdown-tables nbsphinx recommonmark numpydoc
+gpuci_conda_retry create -n cucim -y -c conda-forge -c conda-forge/label/cupy_rc -c rapidsai-nightly \
+    rapids-doc-env \
+    flake8 \
+    pytest \
+    pytest-cov \
+    python=${PYTHON_VER} \
+    conda-forge/label/cupy_rc::cupy=9 \
+    cudatoolkit=${CUDA_VER} \
+    numpy \
+    scipy \
+    scikit-image=0.18.1 \
+    openslide
```

Comment:
are all of these dependencies needed for doc builds?

Reply:
Probably `flake8` and `pytest*` can be dropped. The others are likely still needed as Sphinx `import`s modules